changelogs/2024-10-03_live-configuration-and-data-editing.mdx

```diff
@@ -33,6 +33,8 @@ _inputs:
 
 * CloudCannon now uses the Collection preview options for Select and Multiselect inputs populated by a Collection.
 * You can now configure the full set of preview options for Select and Multiselect inputs, including `icon`, `metadata`, `gallery`, `subtext`, `image`, and `tags`. Additionally, the new keys `view`, `picker_view`, and `picker_preview` are now available.
+* Added the *CloudCannon Configuration Path* field to the *Editing* page under *Site Settings* for specifying the location of your configuration file. You can no longer configure this path in your environment variables.
```

Comment:
This is only relevant for unified config sites — existing sites still set it in env vars.